Wikipedia/Code/ReadingThemesControlsViewController.swift

```diff
@@ -55,6 +57,9 @@ open class WMFReadingThemesControlsViewController: UIViewController {
         
         autoNightModeSwitch.isEnabled = false
         imageDimmingSwitch.isEnabled = false
+        
+        NotificationCenter.default.addObserver(self, selector: #selector(self.screenBrightnessChangedInApp(notification:)), name: NSNotification.Name.UIScreenBrightnessDidChange, object: nil)
```

Comment:
Needs a `NotificationCenter.default.removeObserver(self)` in a `deinit` method (or move add to `willAppear` and remove to `didDisappear`)